### PR TITLE
added cask for abyss web server

### PR DESCRIPTION
**Pull request by freeslugs**
Created cask for abyss web server
tested locally and audited successfully

### CODE REVIEW

File: Casks/abyss.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'abyss' do
+  version 'x1'
+  sha256 '63b0b8d6420a0f65899ea1fa7827f04c8c8b3fa444a9f6361054d6605cf13fed'
+
+  url 'http://www.aprelium.com/data/abwsx1.dmg'
+  name 'Abyss Web Server'
+  homepage 'http://www.aprelium.com/abyssws'
+  license :freemium
```
Comment (vitorgalvao) [inline Casks/abyss.rb]: Is this a free app that works indefinitely but with limitations that can be removed by paying?

Comment (freeslugs) [inline Casks/abyss.rb]: I think it's free. according to the site,
"Free personal edition. It isn't a trial version or a demo. It is a fully usable personal web server with no limitations, no nag screens, no spyware, and no advertisements."

Comment (vitorgalvao) [inline Casks/abyss.rb]: Then it should be `:gratis`, not `:freemium`.

Comment (freeslugs) [inline Casks/abyss.rb]: @vitorgalvao Just updated it.

**Pull request merged**